<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="1" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="58666" started-at="2018-03-31T12:12:00Z" finished-at="2018-03-31T12:12:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="58666" started-at="2018-03-31T12:12:00Z" finished-at="2018-03-31T12:12:59Z">
      <class name="com.omayo.hybrid.testcases.GmailLogin">
        <test-method status="FAIL" signature="doLogin(java.util.Hashtable)[pri:0, instance:com.omayo.hybrid.testcases.GmailLogin@6e0f5f7f]" name="doLogin" duration-ms="55441" started-at="2018-03-31T12:12:00Z" data-provider="getData" finished-at="2018-03-31T12:12:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Name=selenium byarun, Username=seleniumbyarun99@gmail.com, Password=DummyPassword1$, Runmode=Y, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Failure in element extraction: emailLoginTextField_id]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Failure in element extraction: emailLoginTextField_id
at org.testng.Assert.fail(Assert.java:96)
at com.omayo.hybrid.GenericKeywords.getElement(GenericKeywords.java:171)
at com.omayo.hybrid.GenericKeywords.input(GenericKeywords.java:92)
at com.omayo.hybrid.Keywords.executeKeyWords(Keywords.java:58)
at com.omayo.hybrid.testcases.GmailLogin.doLogin(GmailLogin.java:59)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="SKIP" signature="doLogin(java.util.Hashtable)[pri:0, instance:com.omayo.hybrid.testcases.GmailLogin@6e0f5f7f]" name="doLogin" duration-ms="3" started-at="2018-03-31T12:12:56Z" data-provider="getData" finished-at="2018-03-31T12:12:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Name=xyz, Username=U2, Password=P2, Runmode=N, Browser=Chrome}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping as the Runmode is set to N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping as the Runmode is set to N
at com.omayo.hybrid.testcases.GmailLogin.doLogin(GmailLogin.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="SKIP" signature="doLogin(java.util.Hashtable)[pri:0, instance:com.omayo.hybrid.testcases.GmailLogin@6e0f5f7f]" name="doLogin" duration-ms="9" started-at="2018-03-31T12:12:56Z" data-provider="getData" finished-at="2018-03-31T12:12:56Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Name=abc, Username=U3, Password=P3, Runmode=N, Browser=Firefox}]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping as the Runmode is set to N]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping as the Runmode is set to N
at com.omayo.hybrid.testcases.GmailLogin.doLogin(GmailLogin.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLogin -->
        <test-method status="PASS" signature="quit()[pri:0, instance:com.omayo.hybrid.testcases.GmailLogin@6e0f5f7f]" name="quit" is-config="true" duration-ms="3123" started-at="2018-03-31T12:12:56Z" finished-at="2018-03-31T12:12:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- quit -->
      </class> <!-- com.omayo.hybrid.testcases.GmailLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
